x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/follower_indices_table/follower_indices_table.js

```diff
@@ -88,7 +88,10 @@ export const FollowerIndicesTable = injectI18n(
             return isPaused ? (
               <FollowerIndexResumeProvider>
                 {(resumeFollowerIndex) => (
-                  <span onClick={() => resumeFollowerIndex(name)}>
+                  <span
+                    onClick={() => resumeFollowerIndex(name)}
+                    data-test-subj="ccrFollowerIndexListPauseActionButton"
```

Comment:
For these buttons, can we append the name of the index to make it easier to find the right button?

```jsx
data-test-subj={`ccrFollowerIndexListPauseActionButton-${name}`}
```

Reply:
Not an issue in this case since index names can't contain spaces, but if they _could_ than we'd have to do something like Rashmi did: https://github.com/elastic/kibana/pull/30280/files#diff-e2f7db1b8201b14ff0ff5786f6925882R282. Just thought I'd mention it because it jumped out at me in her PR.

Reply:
the line in that PR is actually pulling in the column name, i.e. `jobTableCell-rollupIndex`. the consumption then maps it by row: https://github.com/elastic/kibana/pull/30280/files#diff-69f5d3dbce7f2a92bf05cb7afcf44934R57
so I think we're ok as long as the subjects are unique per row 😄

Reply:
> so I think we're ok as long as the subjects are unique per row

Sorry, I'm not sure I follow... I was trying to say that if there are multiple rows rendered, and we want to perform a pause/resume/edit action on a specific row, we could use the `name` to do so if the `name` is part of the test subject selector for that action. As it is now, someone writing a test couldn't target a specific row's pause action. The best they could do is select all instances of `ccrFollowerIndexListPauseActionButton` and try to infer the correct one based on order.

Reply:
I understand that this subject is not unique since it doesn't have the object's name/ID in it. following the mapping pattern I linked to, there isn't a need for each cell of each row to be unique. the resulting mapped array would contain elements that represent each row of the follower indices table. but as I type this comment I realized that the actions here are a popover, so `ccrFollowerIndexListPauseActionButton` will always be unique anyway

Reply:
Thanks for explaining over Zoom, makes a lot more sense to me now!